components/nexus-repository/src/main/java/org/sonatype/nexus/repository/view/Status.java

```diff
@@ -12,14 +12,21 @@
  */
 package org.sonatype.nexus.repository.view;
 
+import java.io.Serializable;
+
 import javax.annotation.Nullable;
 
+// TODO: implement Externalizable?  This is made Serialzable just for cache usage, may need to revisit to avoid needing
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)